internal-api/src/main/proto/ua/marchenko/internal/commonmodels/auction/auction.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package ua.marchenko.internal.commonmodels.auction;
+
+import "google/protobuf/timestamp.proto";
+import "ua/marchenko/internal/commonmodels/general/big_decimal.proto";
+
+option java_multiple_files = true;
+
+message Auction {
+  string id = 1;
+  string artwork_id = 2;
+  commonmodels.general.BigDecimal startBid = 3;
+  repeated Bid buyers = 4;
```

Comment:
```suggestion
  repeated Bid bid = 4;
```
?

Reply:
I have `buyers` field in my mongo model too, so I write this name in proto to match naming across the project